website/client-old/js/controllers/notificationCtrl.js

```diff
@@ -144,6 +144,11 @@ habitrpg.controller('NotificationCtrl',
           case 'LOGIN_INCENTIVE':
             Notification.showLoginIncentive(User.user, notification.data, Social.loadWidgets);
             break;
+          case 'GROUP_INVITE_ACCEPTED':
```

Comment:
Do we not have a generic notification modal we use when the type isn't found on the client?